server/base/src/main/java/org/apache/accumulo/server/fs/PerTableVolumeChooser.java

```diff
@@ -68,11 +68,6 @@ public String choose(VolumeChooserEnvironment env, String[] options)
   // visible (not private) for testing
   VolumeChooser getDelegateChooser(VolumeChooserEnvironment env) {
     switch (env.getScope()) {
-      case INIT:
-        // TODO should be possible to read from SiteConfiguration during init
-        log.warn("Not possible to determine delegate chooser at '{}' scope. Using {}.",
-            ChooserScope.INIT, RandomVolumeChooser.class.getName());
-        return randomChooser;
```

Comment:
At this point, the switch statement (here and in the other class) could be turned into a simple conditional:

```java
if (env.getScope() == ChooserScope.TABLE) {
    return getVolumeChooserForTable(env);
}
return getVolumeChooserForScope(env);
```

This probably comes down to style also... both work fine, but the conditional is slightly shorter and probably easier to read, since the choice is binary.